core/themes/seven/css/style.css

```diff
@@ -692,6 +692,9 @@ fieldset .fieldset-wrapper {
   border-radius: 4px;
   background: transparent;
 }
+.js[dir="rtl"] fieldset.collapsible .fieldset-title {
+  background-image: none;
```

Comment:
It's not clear to me how or why this would do anything. Is there a background image in the first place that needs hiding?

Reply:
> Is there a background image in the first place that needs hiding?

yes. It is added in system.theme.css:
```
.js[dir="rtl"] fieldset.collapsed .fieldset-title {
    background-image: url(../../../misc/menu-collapsed-rtl.png);
    background-position: 98% 50%;
    background-repeat: no-repeat;
}
``